### PR TITLE
added endpoint /images/reserved-usernames

### PR DESCRIPTION
**Pull request by resoluteCoder**
Add functionality to get reserved usernames on system -> returns an array

### CODE REVIEW

File: pkg/images/images.go [hunk 1]
```diff
@@ -785,3 +786,10 @@ func exeInjectionScript(kickstart string, image string, imageID uint) error {
 	log.Infof("fleetkick output: %s\n", output)
 	return nil
 }
+
+func GetReservedUsernames(w http.ResponseWriter, r *http.Request) {
+	c1, _ := exec.Command("sed", "1d;/^#/d;s/\t.*//", "/usr/share/doc/setup/uidgid").Output()
```
Comment (biancarosa) [inline pkg/images/images.go:791]: I'm just missing an error check here to return an internal server error if this command fails 😬

**Review by biancarosa — CHANGES_REQUESTED** [review]

File: pkg/images/images.go (hunk 1 above)
Comment (resoluteCoder) [reply pkg/images/images.go:791]: Error handling, wooooooops. I'll get on it :smile:

**Comment by maxamillion** [comment]
LGTM :+1:

**Review by biancarosa — APPROVED** [review]
LGTM!

**Pull request merged**